package.json

```diff
@@ -281,6 +281,7 @@
     "@babel/preset-env": "7.8.7",
     "@babel/preset-react": "7.8.3",
     "@babel/preset-typescript": "7.16.5",
+    "@formatjs/icu-messageformat-parser": "2.0.19",
```

Comment:
used to parse strings like `hello this is {name}` and `hello this is {name, select, NONE {nobody} other {{name}}`